README.md

```diff
@@ -97,6 +97,13 @@ Follow this guide to setup your development machine.
 
```

Comment:
This file should not be modified in this PR as it is not required for the issue. Please remove it.